_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+</figure>
+
+If you have ever tried to cross the street in downtown Salt Lake City, you’ll know our intersections are extremely wide. You’ll see the Gunter Chain’s lasting mark at these intersections as well. Looking at Portland (Fig 9) and Salt Lake City (Fig 9) again and you’ll notice two variations of the chain. Salt Lake City (Fig 9) has large intersections that are based on two chain lengths (132 feet), whereas in Portland and San Francisco intersections measure half a chain (33’). In Phoenix (Fig 9), you’ll notice intersections measure one full chain length (66 feet).
+
+## Section Lines and Addressing
+{: .text-left}
+
+Another notable mention are “section line roads”. These roads are laid out in a one-mile by one-mile grid pattern that follow the section lines. Figure 10 shows a good example of this southwest of Boise, Idaho. This pattern is also very popular in Phoenix and Las Vegas.
+
+Section lines in North Dakota and South Dakota are often used as the basis for their street numbering systems. In Salt Lake City, a similar section-based addressing system is used, where the 100 blocks (ex: from 100 E to 200 E) are ten chains, or 660 feet, or quarter-quarter-quarter sections (10 chains or 660 feet).
+
+One last notable Western development pattern is the disguised grid. These are often large communities. The land is typically purchased by the section (or more) and then made to feel less grid-like. Two good examples of this are Sun Lakes (Fig 11) and Sun City in Arizona. As you move through these communities you forget that you're on the grid.
+
+<figure class="caption caption--left caption--flow">
+  <a href="{{ "/images/SectionLineRoads.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/SectionLineRoads_sm.png" | prepend: site.baseurl }}" alt="Section Line Roads Southwest of Boise, ID">
+  </a>
+  <figcaption class="caption__text">Fig 10: Section Line Roads Southwest of Boise, ID</figcaption>
+</figure>
+
+<figure class="caption caption--right caption--flow">
+  <a href="{{ "/images/SunLakes_grid.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/SunLakes_grid_sm.png" | prepend: site.baseurl }}" alt="Sun City, AZ (Disguising the Grid)">
+  </a>
+  <figcaption class="caption__text">Fig 11: Sun City, AZ (disguising the grid)</figcaption>
+</figure>
+
+It's remarkable how a few key points in history have permanently affected the growth and development patterns in the Western United States. It's also interesting how these same decisions have made it difficult for us to turn our backs on the imperial-based measurement system (i.e., United States customary units).
+
+So, the next time you're out wandering the Western grid and you encounter a large intersection or a big city block, you can think back to Edmund Gunter, the Treaty of Paris, and Thomas Jefferson.
```

Comment:
After the article, we'll need to add in the footnotes (as mentioned in prev comments), as well as the sources list. Not sure of the best way to add these in, so here they are! 

American Publishing Company. “Salt Lake City 1891 Bird's Eye View.” _Historic Map Works_, Residential Genealogy. Accessed March 1, 2019. http://www.historicmapworks.com/Atlas/US/9286/Salt+Lake+City+1891+Bird%27s+Eye+View/.

Bonner, John, ca. 1643-1726,  Price, William, fl. 1725-1769, and Dewing, Francis, fl. 1716-1722. “The town of Boston in New England.” Map. 1723. _Norman B. Leventhal Map & Education Center_. Accessed March 01, 2019. https://collections.leventhalmap.org/search/commonwealth:9s161f21f.

Bunce, Greg. “Variations on the Western Urban Grid.” _Automated Geographic Reference Center_. March 01, 2019.  

Golbez. “States and Territories of the United States of America August 7 1789 to April 2 1790.” _Wikipedia_, “Northwest Territory” page. Accessed March 1, 2019.

Google Maps. “Boise Idaho.” Accessed March 01, 2019. https://www.google.com/maps/place/Boise,+ID/@43.6088162,-116.3498057,7348m/data=!3m1!1e3!4m5!3m4!1s0x54aef172e947b49d:0x9a5b989b36679d9b!8m2!3d43.6150186!4d-116.2023137.

Google Maps. “Sun Lakes, Arizona.” Accessed March 01, 2019. https://www.google.com/maps/place/Sun+Lakes,+AZ+85248/@33.2181668,-111.8759392,3672m/data=!3m1!1e3!4m5!3m4!1s0x872b000a3cf645db:0xeb2fbae417540df7!8m2!3d33.2181797!4d-111.8759103. 

“Great Salt Lake Meridian/Base Marker.” _HowderFamily.com_. Updated August 13, 2013. Accessed March 01, 2019. https://www.howderfamily.com/travel/utah/great-salt-lake-base-and-meridian.html.

“Gunter Chain.” (Article.)  _Smithsonian National Museum of American History_. Accessed March 01, 2019. http://americanhistory.si.edu/collections/search/object/nmah_761634.  

“Gunter Chain.” (Image.) John Johnson (owner). _Smithsonian National Museum of American History_. Accessed March 01, 2019. http://americanhistory.si.edu/collections/search/object/nmah_761634.  

“Projections and Survey Systems.” (Article.) _San Francisco Estuary Institute & The Aquatic Science Center_. Accessed March 01, 2019. https://www.sfei.org/it/gis/map-interpretation/projections-and-survey-systems#sthash.D2D94lF3.sWlKZBU3.dpbs.

“Revised Plat of the City of Zion, circa Early August 1833.” p. [1]. _The Church of Jesus Christ of Latter-day Saints, The Joseph Smith Papers_. Accessed March 01, 2019. https://www.josephsmithpapers.org/paper-summary/revised-plat-of-the-city-of-zion-circa-early-august-1833/1.

Sellers, Christine and Martin, James. “The Articles of Confederation: The First Constitution of the United States.” _Library of Congress_. Published September 16, 2011. Accessed March 01, 2019. https://blogs.loc.gov/law/2011/09/the-articles-of-confederation-the-first-constitution-of-the-united-states/.    

United States Bureau of Land Management. “Principal Meridians and Base Lines.” (Image.) _US Geological Survey_. Updated January 18, 2018. Accessed March 01, 2019.  https://nationalmap.gov/small_scale/a_plss.html.

United States Bureau of Land Management. “The Public Land Survey System (PLSS).” (Article.) _US Geological Survey_. Updated January 18, 2018. Accessed March 01, 2019.  https://nationalmap.gov/small_scale/a_plss.html.

“US Public Land Survey System.” (Image.) _San Francisco Estuary Institute & The Aquatic Science Center_. Accessed March 01, 2019. https://www.sfei.org/it/gis/map-interpretation/projections-and-survey-systems#sthash.D2D94lF3.sWlKZBU3.dpbs.

Williams, Frederick G. “Revised Plat of the City of Zion, Circa Early August 1833.” _The Church of Jesus Christ of Latter-day Saints, The Joseph Smith Papers_. Updated September 01, 2018. Accessed March 01, 2019. https://www.josephsmithpapers.org/paper-summary/revised-plat-of-the-city-of-zion-circa-early-august-1833/1.

Wright, Bryan. “Gunter’s Chain: Early Surveying in America.” _Colonial Sense_. Accessed March 01, 2019. http://www.colonialsense.com/Society-Lifestyle/Signs_of_the_Times/Gunter%27s_Chain.php.